meetings.md

```diff
@@ -0,0 +1,65 @@
+# Meetings
+
+The purpose of this resource is to share details about how and when we
+meet.
+
+**Contents**
+- [Types](#types)
+  - [All Hands](#all-hands)
+    - [Standup](#standup)
+    - [Full](#full)
+  - [Holds](#holds)
+- [Scheduling](#scheduling)
+- [Hosting](#hosting)
+- [Expectations](#expectations)
+
+## Types
+
+### All Hands
+
+All-hands meetings are our general sync-up. We all try our best to be
+there. We adjust regularity of meeting based on time of year, member
+workload (inside and outside co-op), and needs of organization-wide
+initiatives.
+
+:calendar: **Schedule:** We currently meet every 2 weeks, alternating between
+format of a short **standup** and a **full** meeting.
+
+#### Standup
+:clock10: **Duration:** 15 minutes
+
+This format involves...
+
+#### Full
+:clock10: **Duration:** 60 minutes
+
+This format involves...
+
+### Holds
+
+Holds are unclaimed spots in our shared schedule that we try to leave
+open. They could be used for:
+
+- Discussions requiring the whole organization
+- Last-minute or urgent meetings, for which a new scheduling is
+  difficult
+
+## Scheduling
```

Comment:
This seems confusing/long to have a heading that will just direct to another area, could this + hosting be made one comment at the top?

"See "Sched" and "Hosting" meeting protocols" for specific instructions

Reply:
Good pattern.